apps/prairielearn/src/migrations/20240104213403_saml_providers__add_additional_config.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE saml_providers
+ADD COLUMN validate_audience boolean NOT NULL DEFAULT false;
+
+ALTER TABLE saml_providers
+ADD COLUMN want_assertions_signed boolean NOT NULL DEFAULT false;
+
+ALTER TABLE saml_providers
+ADD COLUMN want_authn_response_signed boolean NOT NULL DEFAULT false;
```

Comment:
Maybe we should follow these ALTER commands with an additional set to make the DEFAULT be TRUE, as "documentation" for what we want here? I know it won't make any actual difference to the current code execution but I like having the DB defaults be what we want.

Reply:
Done in bd770f6.